qutebrowser/config/configdata.yml

```
@@ -1173,6 +1173,44 @@ editor.encoding:
   default: utf-8
   desc: Encoding to use for the editor.
 
+## fileselect
+
+fileselect.handler:
+  default: default
+  type:
+    name: String
+    valid_values:
+      - default: "Use the default file selector."
+      - external: "Use an external command."
+  desc: Handler for selecting file(s) in forms.
+
+fileselect.single_file.command:
+  type:
+    name: ShellCommand
+    placeholder: true
+  default: ['xterm', '-e', 'export COLUMNS= LINES=;ranger --choosefile={}']
+  desc: >-
+    Command (and arguments) to use for selecting a file in forms.
```

```suggestion
    Command (and arguments) to use for selecting a single file in forms.
```

To make it a bit clearer maybe?